src/line.rs

```diff
@@ -17,7 +16,7 @@ pub struct Line {
     pub text: Option<String>,
     #[serde(rename="cursor")]
     pub cursors: Option<Vec<u64>>,
-    pub styles: Option<Vec<i64>>,
+    pub styles: Vec<i64>,
```

Comment:
is styles guaranteed to always be present? last I checked the protocol, that was not the case